.travis.yml

```diff
@@ -1,27 +1,45 @@
 language: python
-
-env:
```

Comment:
I think you can keep the old, shorter, version by adding `dist: xenial` at the top level, as in https://github.com/itamarst/eliot/blob/master/.travis.yml#L2

Reply:
Okay, I'll give that a shot. I *think* it's going to mash the two Twisted versions together in a single sub-test unless I pull the Twisted selection stuff back out of the Tox setup (which would make it a little harder to run the same tests locally), but I'll see.

Reply:
Ah, then maybe also do what I did in Eliot and use `tox-travis`?

Make sure tox-travis is installed:
https://github.com/itamarst/eliot/blob/master/.travis.yml#L9

Add a mapping from Travis Python version to the tox commands to run:
https://github.com/itamarst/eliot/blob/master/tox.ini#L6